src/ape/managers/chain.py

```diff
@@ -369,14 +368,12 @@ def pending_timestamp(self) -> int:
             chain.pending_timestamp += 3600
         """
 
-        pending_block = self.provider.get_block("pending")
-        return int(pending_block.timestamp + self._time_offset)
+        return int(self.provider.get_block("pending").timestamp)
```

Comment:
is this not an int by default?

Reply:
it is a float, I don't get why though because I think it actually is just always an int

Reply:
Okay, yeah this should be an int: https://ethereum.stackexchange.com/questions/7853/is-the-block-timestamp-value-in-solidity-seconds-or-milliseconds/7854